arch/arm/src/lpc17xx_40xx/lpc17_40_sdcard.c

```diff
@@ -317,7 +322,8 @@ static inline uint32_t lpc17_40_getpwrctrl(void);
 static void lpc17_40_sampleinit(void);
 static void lpc17_40_sdcard_sample(struct lpc17_40_sdcard_regs_s *regs);
 static void lpc17_40_sample(struct lpc17_40_dev_s *priv, int index);
-static void lpc17_40_sdcard_dump(struct lpc17_40_sdcard_regs_s *regs, const char *msg);
+static void lpc17_40_sdcard_dump(struct lpc17_40_sdcard_regs_s *regs,
+                                 const char *msg);
```

Comment:
Indentation style is not consistent.  In all other prototypes, the argument that continue on the same coleumn.  Like the following.

Reply:
Hmm, I missed the column start position, normally I use as reference the "(" in the previous line. But some cases like ABIGMACRONAMEA | ABIGMACRONAMEB I try to keep it aligned the beginning of the first macro name. I will fix it